shared/src/main/scala/squants/space/Angle.scala

```diff
@@ -34,6 +35,10 @@ final class Angle private (val value: Double, val unit: AngleUnit)
   def tan = math.tan(toRadians)
   def asin = math.asin(toRadians)
   def acos = math.acos(toRadians)
+
+  protected def timeDerived: AngularVelocity = RadiansPerSecond(toRadians)
```

Comment:
I wonder why on `AngularVelocity` the protected methods are `[squants]` protected but not in `Angle`